examples/12-fluids/00-fluids_model.py

```diff
@@ -0,0 +1,193 @@
+"""
+.. _ref_fluids_model:
+
+Explore Fluids models
+------------------------------------------------------
+
+"""
+
+###############################################################################
+# Exploring an Ansys Fluent model
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# This example demonstrates how you can explore an Ansys Fluent model. Import
+# the result file and create a model
+
+import ansys.dpf.core as dpf
+from ansys.dpf.core import examples
+
+path = examples.download_fluent_axial_comp()["flprj"]
+ds = dpf.DataSources(path)
+model = dpf.Model(data_sources=ds)
+
+
+###############################################################################
+# Exploring the mesh
+# ------------------
+# Explore the mesh through the MeshInfo. The MeshInfo provides metadata information
+# about the mesh. For fluid models, it is useful to know the cell and face zones,
+# as well as the topological relationships between them. First get all the available
+# information in the MeshInfo
+
+minfo = model.metadata.mesh_info
+print(
+    "\n".join(
+        "{}\t{}".format(k, v)
+        for k, v in minfo.generic_data_container.get_property_description().items()
+    )
```

Comment:
@rafacanton shouldn't we create a helper for that if that is something people are going to want to do?

Reply:
I agree, I'm adding it